llvmlite/tests/test_ir.py

```diff
@@ -2225,6 +2225,23 @@ def test_bitcast(self):
         c = gv.bitcast(int64.as_pointer())
         self.assertEqual(str(c), 'bitcast (i32* @"myconstant" to i64*)')
 
+    def test_ptrtoint(self):
+        m = self.module()
+        gv = ir.GlobalVariable(m, int32, "myconstant")
+        c = gv.ptrtoint(int64)
+        self.assertEqual(str(c), 'ptrtoint (i32* @"myconstant" to i64)')
+
+        self.assertRaises(
+            TypeError,
+            gv.ptrtoint,
+            int64.as_pointer())
+
+        c2 = ir.Constant(int32, 0)
+        self.assertRaises(
```

Comment:
Suggest using `assertRaisesRegex` to check the exception string is that intended.

Reply:
Done